cloudimg/aws.py

```diff
@@ -699,9 +716,7 @@ def tag_image(self, image, tags):
                 } for k, v in tags.items()
             ],
         }
-
         res = image.create_tags(**attrs)
-        log.debug("Tag image \"%s\" results: %s" % image.name, res)
         return res
```

Comment:
res is a class and was throwing errors writing to log.

Reply:
Ok, got it!

Thanks for letting me know

Reply:
I'm a little confused about how the earlier code even worked because it was mixing "explicitly format the string" and "let logger format the string" styles.

Trying something like this myself, it indeed crashes:

```python
>>> logging.getLogger().debug("foo %s results: %s" % 'bar', 'baz')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: not enough arguments for format string
```

Unless image.name happens to be a tuple, which I suppose is possible, then it seems like this should also be crashing since it takes a string with two "%s" and then provides it with only a single value.

The correct way to write the code would have been:

```python
log.debug("Tag image \"%s\" results: %s", image.name, res)
```

Just to be sure, the error you were seeing is unrelated to this?  On the surface of it there doesn't seem to be a reason why str() or repr() on `res` should fail.